types/testingutils/keymanager.go

```diff
@@ -193,12 +208,24 @@ func (km *testingKeyManager) Decrypt(pk *rsa.PublicKey, cipher []byte) ([]byte,
 
 // Encrypt given a rsa pubkey and data returns an PKCS1v15 e
 func (km *testingKeyManager) Encrypt(pk *rsa.PublicKey, data []byte) ([]byte, error) {
-	panic("implement")
+	return FakeEncryption(data), nil
```

Comment:
shouldn't be fake, the keys can be hardcoded but better use the real encryption for test correctness

Reply:
Changed to use types.Encrypt.